internal/build/path_mapping.go

```diff
@@ -81,35 +84,24 @@ func FilterMappings(mappings []pathMapping, matcher model.PathMatcher) ([]pathMa
 }
 
 // FilesToPathMappings converts a list of absolute local filepaths into pathMappings (i.e.
-// associates local filepaths with their mounts and destination paths).
-func FilesToPathMappings(files []string, mounts []model.Mount) ([]pathMapping, error) {
-	pms, err := filesToPathMappings(files, mounts)
-	if err != nil {
-		return pms, err
-	}
-	return pms, nil
-}
-
-func filesToPathMappings(files []string, mounts []model.Mount) ([]pathMapping, *PathMappingErr) {
+// associates local filepaths with their mounts and destination paths). If a file does
+// not belong to a mapping, ignore it.
```

Comment:
this behavior change seems wrong? this has caught a lot of legitimate problems. can you give an example of a case where we wouldn't want this to be an error?

Reply:
aah I was imagining passing a Tiltfile along as a changed file -- doesn't belong to a mount, but that's an expected case. I didn't realize this had caught legitimate problems tho, I thought this case never got tripped b/c of the way we set up watches. will rework.